### DISCUSSION TITLE
Toggle rendering script options for individual files

### DISCUSSION THREAD

**Discussion by juliantao** [quarto-dev/quarto-cli#4084] [Q&A] [answered]
Currently, "Pre and post render scripts are run with the main project directory." It is very handy for project-wide rendering. 

But from time to time, I need to enable and disable a script for a working file when previewing. It will be very handy if one can include `post-render: ../scripts/post.py` or `post-render: false` to the `yaml` header to toggle the option on and off. 

Or are there any other ways to achieve the goal?

**Comment by cscheid** [comment, marked as answer]
I think project profiles would work: https://quarto.org/docs/projects/profiles.html

**Comment by juliantao** [comment]
Thank you, @cscheid. It works!
